### PR TITLE
Markdown test

### PR DESCRIPTION
**Pull request by shunjikonishi**
Implement markdownTest.

@da0shi review me

### CODE REVIEW

File: src/utils/markdownTest.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+function MarkdownTest(answers) {
+  function normalizeArray(array) {
+    for (var i=0; i<array.length; i++) {
+      var v = array[i];
+      if (!util.isArray(v)) {
+        array[i] = [v];
+      }
+    }
+  }
```
Comment (da0shi) [inline src/utils/markdownTest.js]: I think you use normailzeArray in different code.
Maybe it is better to define in util class.

File: src/utils/markdownTest.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  function isCheckbox(str) {
+    if (str.indexOf("- [ ]") === 0) {
+      return true;
+    }
+    if (str.indexOf("- [x]") === 0) {
+      return true;
+    }
+
+    return false;
+  }
```
Comment (da0shi) [inline src/utils/markdownTest.js]: there might be a nested check box like below
```
- some title
  - [ ] check
- some second title
  - [ ] box
    - [ ] here
```
I think it's better to do `str.trim()` before check indexOf

**Comment by da0shi** [comment]
I've added some notes, but mainly
LGTM

**Comment by shunjikonishi** [comment]
@da0shi review again

- Move normalizeArray function to outside of MarkdownTest. If necessary, I will move it to utils.js.
- Add trim to support subList.

**Comment by da0shi** [comment]
lgtm

**Pull request merged**